### DISCUSSION TITLE
Need help with new nanoELS pcb design?

### DISCUSSION THREAD

**Discussion by AlyoLaz** [kachurovskiy/nanoels#128] [General]
Hello 
I have been working in a new PCB design for my nanoELS project. The existing PCB doesn’t work for me because I have split up the PCB (PCB with Arduino) and separate DIY keyboard and moved the display. The schematic stayed the same but I had redrawn it. So I would ask for your help for any tips and mistakes that I might have made while redesigning it, before I order the PCB from JLC PCB. 
Thank you for your help!
When I am done I will be posting the whole nanoELS project …. It will be soon. This PCB is the last thing.
![Schematic_Pcb Test_2023-08-12](https://github.com/kachurovskiy/nanoels/assets/142013893/b4fb882b-5113-4f3c-bff9-b3859a841029)
![Schematic_KEYBOARD_2023-08-12](https://github.com/kachurovskiy/nanoels/assets/142013893/001ab3f6-4229-4d02-997d-017499c37d50)
![pcb1](https://github.com/kachurovskiy/nanoels/assets/142013893/671d4552-7ec6-4f1f-8f88-181eb78f8a45)
![pcb2](https://github.com/kachurovskiy/nanoels/assets/142013893/339e551c-f97e-4653-9a27-e6994b3a9972)
![20230812_185759](https://github.com/kachurovskiy/nanoels/assets/142013893/d5e6846d-d74e-43b2-9015-062c847be002)

**Comment by kachurovskiy** [comment]
Hi @AlyoLaz, awesome project! It's really hard to chase all those lines visually so I can't comment as to whether it will work - but the good news is that 2-layer PCBs like this are cheap.

**Comment by AlyoLaz** [comment]
Hi @kachurovskiy, thanks for the response. I hope that it will work lol. I cant wait to share the whole project once I am done.

**Reply by kachurovskiy** [reply]
How's it been going? Did it work out?

**Reply by AlyoLaz** [reply]
Hello @kachurovskiy , sorry for the late response. Fortunately, the pcb does works very nicely. There is no interference! The keyboard cable is like 8cm long and the LCD cable in like 20cm long. The code works nicely, I have 400 pulse/rev setting in the stepper driver and it does everything up to like 800 rpm. This project is actually my college diploma …. I think that how you say it in English, I am actually from Slovenia. So thank you very much without you this would not be possible! When I am finished writing my diploma I will release the whole project on your Nano els page.
